basic-steps/src/main/java/org/jenkinsci/plugins/workflow/steps/TimeoutStep.java

```diff
@@ -61,7 +61,7 @@ public boolean takesImplicitBlockArgument() {
 
         @Override
         public String getDisplayName() {
-            return "Executes the body with a timeout";
+            return "Execution with timeout";
```

Comment:
Maybe `Enforce time limit`? (_Execution_ in English is pretty vague, even covering capital punishment!)